### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by Tristyn**

### CODE REVIEW

File: libraries/ts/src/margin/accountPosition.ts [hunk 1]
```diff
@@ -39,25 +39,15 @@ export class AccountPosition {
   balanceTimestamp: BN
 
   /** The current price/value of each token */
-  price: PriceInfo
+  priceRaw: PriceInfo
+
+  get price(): Number128 {
```
Comment (davodesign84) [inline libraries/ts/src/margin/accountPosition.ts:44]: Do we know if this is used in application code?

File: libraries/ts/src/margin/pool/poolManager.ts [hunk 2]
```diff
@@ -225,7 +225,7 @@ export class PoolManager {
     pythProduct: Address
     pythPrice: Address
     marginPoolConfig: MarginPoolConfigData
-    addresses: MarginPoolAddresses
+    addresses: PoolAddresses
```
Comment (davodesign84) [inline libraries/ts/src/margin/pool/poolManager.ts:228]: Also this, might be used in application code?

**Review by davodesign84 — APPROVED** [review]

File: libraries/ts/src/margin/pool/poolManager.ts (hunk 2 above)
Comment (Tristyn) [reply libraries/ts/src/margin/pool/poolManager.ts:228]: Nope

Comment (davodesign84) [reply libraries/ts/src/margin/pool/poolManager.ts:228]: Cool beans!

File: libraries/ts/src/margin/accountPosition.ts (hunk 1 above)
Comment (Tristyn) [reply libraries/ts/src/margin/accountPosition.ts:44]: Nope pretty sure

**Review by jetjake — APPROVED** [review]

**Pull request merged**